src/utils/changeEventProducer.js

```diff
@@ -537,40 +341,13 @@ class ChangeEventProducer {
                 );
             }
         }
-        if (resourceType === 'Task') {
```

Comment:
We don't need to emit Task events. The task-service already emits those.
But we do need to emit Consent events so we can update Iterable user profiles at the grain of the client Person